cove_360/locale/es/LC_MESSAGES/django.po

```diff
@@ -542,15 +545,19 @@ msgstr ""
 "concedida entre <strong>%(start_date)s</strong> y "
 "<strong>%(end_date)s</strong>."
 
-#: cove_360/templates/cove_360/explore.html:54
+#: cove_360/templates/cove_360/explore.html:107
+msgid "The file was "
+msgstr ""
```

Comment:
Where did this come from with no Translation?

Reply:
This string is only in 360 CoVE, which we don't translate.